fastlane/Scanfile

```diff
@@ -1,14 +0,0 @@
-# For more information about this configuration visit
-# https://docs.fastlane.tools/actions/scan/#scanfile
-
-# In general, you can use the options available
-# fastlane scan --help
-
-scheme("RevenueCat")
-
-# Commenting out devices in Scanfile as of fastlane 2.191.0
-# Scan cannot handle running multiple platform type (iOS and tvOS)
-# in the same run but might in the future
-# Future version of fastlane might allow this
-
-# devices(["iPhone 6", "iPhone 12", "Apple TV"])
```

Comment:
This is not needed anymore because of [this](https://github.com/RevenueCat/purchases-ios/pull/845/files#diff-dff4b99d4e651ab9d7597876ec8dbe92aa934e42c0fb55f4aadd6c106fc96688R52) change right?

Reply:
yep